changelogs/unreleased/fiber-parent-backtrace.md

```diff
@@ -0,0 +1,5 @@
+## feature/core
+
+* Introduce new `fiber.parent_backtrace_enable` and
+  `fiber.parent_backtrace_disable` handles for toggling collection of parent
+  backtraces for newly created fibers (disabled by default).
```

Comment:
Nit: A changelog should be in Past Simple. The ticket number is missing. Better emphasize the feature you introduced, not the handles:

```
* Implemented collection of parent backtrace for newly created fibers.
  To enable the feature, call `fiber.parent_backtrace_enable` (gh-4302).
```

Also, please improve the doc bot request: write a code example for tech writers.

Reply:
Fixed: 
* changed the changelog tense to _Past Simple_;
* improved the changelog to emphasize the parent backtrace collection feature;
* doc bot request now consists of code supplied with comments.

If the doc bot request still doesn't satisfy you, I would be grateful if you provided  a relevant example.